policies/queries/iam/policies_unused.sql

```diff
@@ -0,0 +1,12 @@
+insert into aws_policy_results
+select :'execution_time'                       as execution_time,
+       :'framework'                            as framework,
+       :'check_id'                             as check_id,
+       'Unused IAM policies should be removed' as title,
+       account_id,
+       id                                      as resource_id,
```

Comment:
This field does not exist in `aws_iam_user_policies`

Reply:
changed to aws_iam_policies